src/main/java/uk/gov/hmcts/reform/coh/controller/QuestionRoundController.java

```diff
@@ -61,6 +64,9 @@ public ResponseEntity getQuestionRounds(@PathVariable UUID onlineHearingId) {
         onlineHearing = optionalOnlineHearing.get();
         List<QuestionRound> questionRounds = questionRoundService.getAllQuestionRounds(onlineHearing);
 
```

Comment:
This is a workaround for a Hibernate bug that it can't handle multi bag in some scenarios. There's bound to be a better solution than this. It'll have to do for now.